rules/awsrules/aws_launch_configuration_invalid_iam_profile.go

```diff
@@ -0,0 +1,89 @@
+package awsrules
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/hashicorp/hcl2/hcl"
+	"github.com/wata727/tflint/issue"
+	"github.com/wata727/tflint/tflint"
+)
+
+// AwsLaunchConfigurationInvalidIAMProfileRule checks whether profile actually exists
+type AwsLaunchConfigurationInvalidIAMProfileRule struct {
+	resourceType  string
+	attributeName string
+	profiles      map[string]bool
+	dataPrepared  bool
+}
+
+// NewAwsLaunchConfigurationInvalidIAMProfileRule returns new rule with default attributes
+func NewAwsLaunchConfigurationInvalidIAMProfileRule() *AwsLaunchConfigurationInvalidIAMProfileRule {
+	return &AwsLaunchConfigurationInvalidIAMProfileRule{
+    resourceType:  "aws_launch_configuration",
```

Comment:
[nitpicks] It seems that the indentation is broken. It might be better to run `gofmt` when saving Go files 🙃